package.json

```diff
@@ -73,7 +73,7 @@
     ]
   },
   "overrides": {
-    "minimist": "1.2.5"
+    "minimist": "1.2.6"
```

Comment:
We have got overrides here https://github.com/alphagov/pay-frontend/blob/master/package.json#L76 so they should be updated too